### PR TITLE
Small PR to  expose boost static option

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
all in title

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

**Comment by copyme** [comment]
@kerautret many thanks!

**Comment by kerautret** [comment]
auto merge (small PR) ;)

**Comment by copyme** [comment]
@kerautret maybe small but I was awaiting if for years :D

**Comment by kerautret** [comment]
🎉😃

File: ChangeLog.md [hunk 1]
```diff
@@ -12,7 +12,10 @@
       [#347](https://github.com/DGtal-team/DGtalTools/pull/347))
     - Fix Documentation nightly update on github website.
       (Bertrand Kerautret
-      [#348](https://github.com/DGtal-team/DGtalTools/pull/347))
+      [#348](https://github.com/DGtal-team/DGtalTools/pull/348))
+    - CMake expose boost static option.
```
Comment (dcoeurjo) [inline ChangeLog.md]: Exposes

Comment (kerautret) [reply ChangeLog.md]: thanks changing it on master

**Pull request merged**